sunpy/visualization/animator/tests/test_basefuncanimator.py

```diff
@@ -165,8 +169,8 @@ def test_sanitize_axis_ranges(axis_ranges, exp_extent, exp_axis_ranges):
     out_axis_ranges, out_extent = aanim._sanitize_axis_ranges(axis_ranges=axis_ranges,
                                                               data_shape=data_shape)
     assert exp_extent == out_extent
-    assert np.array_equal(exp_axis_ranges[0], out_axis_ranges[0])
     assert np.array_equal(exp_axis_ranges[1], out_axis_ranges[1])
+    assert callable(out_axis_ranges[0])
```

Comment:
Even though this is a callable now, can we still do a comparision with expected ranges (as in the old line 168 which has now gone)?